### DISCUSSION TITLE
Outlaw Blind Cooldown Condition Not Working

### DISCUSSION THREAD

**Discussion by KaoReal** [Hekili/hekili#1598] [General]
When a condition cooldown.blind.ready is added to using Flagellation, the addon will always read False for this condition regardless of current blind cooldown. Using cooldown.kidney_shot.ready for comparison will work as expected.

**Comment by Hekili** [comment]
Did you add Blind to a toggle?  Is that toggle turned off?

**Reply by KaoReal** [reply]
Blind is on cooldowns and toggle is on. Should it be working with this setup?

**Reply by KaoReal** [reply]
Hmm, I set Blind to toggle = none, and it's working now. Is there any way to keep Blind as a cooldown and have this work? I like having blind in a separate display for cooldowns.

**Reply by Hekili** [reply]
Are your Cooldowns shown separately?  If so, Blind is not "ready" in any other display besides Cooldowns.

You could:
* Move Flagellation to Cooldowns toggle instead of Covenants, and then it would be recommended in Cooldowns display and would know Blind's actual cooldown.
* Use `cooldown.blind.true_remains=0` in place of `.ready` if you keep Flagellation as-is.

**Reply by KaoReal** [reply]
Awesome, thank you! Love the addon, by the way!
